components/brave_wallet/browser/eth_response_parser.cc

```diff
@@ -187,7 +188,9 @@ absl::optional<std::string> ParseEthCall(const base::Value& json_value) {
   return ParseSingleStringResult(json_value);
 }
 
-absl::optional<std::vector<std::string>> DecodeEthCallResponse(
+absl::optional<std::vector<std::string>>
+DecodeEthCallResponse(  // returns "args" a list of decoded values corresponding
+                        // to the types list
```

Comment:
Please move comment before function definition

Reply:
I think I'll just remove this change since I didn't write this function, and the comment was mostly for my reference when coding.